tools/AzPreview/AzPreview.psd1

```diff
@@ -79,7 +79,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.11.2'; },
             @{ModuleName = 'Az.Compute'; RequiredVersion = '5.4.0'; }, 
             @{ModuleName = 'Az.ConfidentialLedger'; RequiredVersion = '1.0.0'; }, 
             @{ModuleName = 'Az.Confluent'; RequiredVersion = '0.2.0'; }, 
-            @{ModuleName = 'Az.ConnectedKubernetes'; RequiredVersion = '0.7.0'; }, 
```

Comment:
Please remove the changes for files under tools folder

Reply:
Sorry. The changes under tools are recovered.